retrieval/target.go

```diff
@@ -197,10 +197,16 @@ func (t *Target) Update(cfg *config.ScrapeConfig, baseLabels, metaLabels clientm
 
 	t.url.Scheme = cfg.Scheme
 	t.url.Path = string(baseLabels[clientmodel.MetricsPathLabel])
+	params := url.Values{}
+	for k, v := range baseLabels {
+		if strings.HasPrefix(string(k), "__param_") {
```

Comment:
This should probably be defined as a constant along with the other label prefixes in https://github.com/prometheus/client_golang/blob/master/model/labelname.go#L23-L63. And yeah, that should be moved out of the client library at some point.